### PR TITLE
last

### PR DESCRIPTION
**Pull request by VasilPan**
- [DEMO LINK](https://VasilPan.github.io/layout_html-form/)
- [TEST REPORT LINK](https://VasilPan.github.io/layout_html-form/report/html_report/)

### CODE REVIEW

**Review by stanislav-topikha — CHANGES_REQUESTED** [review]
remove white spaces from textarea
image

fix this 

https://user-images.githubusercontent.com/99331793/197868160-47c93144-6fa9-4167-b38b-114c7b6559fa.mov

File: src/index.html [hunk 1]
```diff
@@ -4,14 +4,144 @@
...
+    <form action ="https://mate-academy-form-lesson.herokuapp.com/create-application" method="post">
+      <fieldset class="fieldset ">
+        <legend>Personal information</legend>
+        <label  class="label">
+          Surname:
+          <input
+          type="text"
+          name="surname"
+          >
+        </label>
```
Comment (SanyaBratashchuk) [inline src/index.html:23]:
```suggestion
          <input
            type="text"
            name="surname"
          >
        </label>
```

File: src/index.html [hunk 2]
```diff
@@ -4,14 +4,144 @@
...
+        <label class="label">
+          Name:
+          <input
+          type="text"
+          name="name"
+          >
```
Comment (SanyaBratashchuk) [inline src/index.html:29]:
```suggestion
          <input
            type="text"
            name="name"
          >
```

File: src/index.html [hunk 3]
```diff
@@ -4,14 +4,144 @@
...
+        <label name="perinfo" class="label">
+          How old are You?
+          <input
+          type="number"
+          name="age"
+          min="0"
+          max="100"
+          value="12"
+          autocomplete="off"
+          >
```
Comment (SanyaBratashchuk) [inline src/index.html:40]:
```suggestion
          <input
            type="number"
            name="age"
            min="0"
            max="100"
            value="12"
            autocomplete="off"
          >
```

**Review by SanyaBratashchuk — CHANGES_REQUESTED** [review]
Check your codestyle with [checklist](https://github.com/mate-academy/layout_html-form/blob/master/checklist.md)

**Pull request closed without merging**